### ISSUE TITLE
Document the `winesapos-convert` script

### ISSUE THREAD

**Issue by GuestSneezeOSDev** [LukeShortCloud/winesapOS#929]
**Describe the bug or feature**
Document the Conversion script
**Issue Type**
- [ ] Bug Report
- [ ] Feature Request
- [x] Documentation
- [ ] Question
- [ ] Other [Please specify]

**To Reproduce**
1. 

**Expected behavior**
Not an issue
**Screenshots**
Not an issue
**System information (please complete the following information):**
- Version: [4.1.0]
- Image type: [Minimal]
- Computer type: [PC]
- Graphics vendor: [Intel]
- Graphics card: [Intel Haswell ULT]
- Drive type: [NVMe]
- Drive connection to PC: [USB]

**Additional context**

**Comment by LukeShortCloud** [comment]
I made some few last minute chances to the conversion script:
- Checks were added in case a user re-runs the script.
- If the system is not Arch Linux or Manjaro, at least Flatpaks will be installed.
- All AUR packages used by winesapOS have been added to the script.

I'm finishing some last minute testing but it appears to be working. I'll merge that and all the relevant documentation soon. Then I'll finally tag the 4.1.0 release and make a public announcement about it. :-)
